packages/filebrowser/src/listing.ts

```diff
@@ -1138,6 +1150,10 @@ export class DirListing extends Widget {
       document.addEventListener('mouseup', this, true);
       document.addEventListener('mousemove', this, true);
     }
+
+    if (this.getAllowSingleClickNavigation()) {
```

Comment:
Allows to reduce the API surface, and removes one function call :)

```suggestion
    if (this._allowSingleClick) {
```